csharp/NEP17/NEP17.Owner.cs

```diff
@@ -0,0 +1,46 @@
+using Neo.SmartContract.Framework;
+using Neo.SmartContract.Framework.Services.Neo;
+using System;
+
+namespace Neo.SmartContract.Examples
+{
+    partial class NEP17Demo
+    {
+        public static void _deploy(object data, bool update)
+        {
+            if (update) return;
+            if (TotalSupplyStorage.Get() > 0) throw new Exception("Contract has been deployed.");
+
+            TotalSupplyStorage.Increase(InitialSupply);
```

Comment:
Use mint?

Reply:
What does it mean?

Reply:
@shargon `Mint` is for crowd sale I think.